### DISCUSSION TITLE
Can Get nut can't Set with pnp.powershell

### DISCUSSION THREAD

**Discussion by RichWorld-Tech** [pnp/powershell#248] [General]
I am a Global Admin and everything was working fine using the  SharepointPnPPowerShellOnline Module. I loaded the pnp.powershell module, went through the process and the the PnP Management Shell Enterprise App is out there with all its permissions. I verified I am connecting to the App and can use GET commands all day long but I cannot use SET commands and receive the following errors. Any assistance would be greatly appreciated.   

Set-pnpuserprofileproperty : Access denied. You do not have permission to perform this action or access this resource.
At line:1 char:1
+ Set-pnpuserprofileproperty -account toriano.evans@baycare.org -proper ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : WriteError: (:) [Set-PnPUserProfileProperty], ServerUnauthorizedAccessException
    + FullyQualifiedErrorId : EXCEPTION,PnP.PowerShell.Commands.UserProfiles.SetUserProfileProperty

**Comment by erwinvanhunen** [comment]
Hi, none of the set-* cmdlets work? Or just this one? We modified the PnP Management Shell application a last week ago to actually include the permissions for the user profile service. If you went through the granting the PnP Management Shell application before that you would have to do this again. Just launch `Register-PnPManagementShell` again.

**Comment by thuld** [comment]
Seems like I am facing the same problem. I have registered the application today (PnP.PowerShell version 1.3.0)

**Code**
`Set-PnPUserProfileProperty -Account 'test@foobar.com' -PropertyName 'WorkPhone' -Value '0123456789'`

**Error:** 
````
Set-PnPUserProfileProperty : Access denied. You do not have permission to perform this action or access this resource.
At line:1 char:1
+ Set-PnPUserProfileProperty -Account 'test@foobar.com ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : WriteError: (:) [Set-PnPUserProfileProperty], ServerUnauthorizedAccessException
    + FullyQualifiedErrorId : EXCEPTION,PnP.PowerShell.Commands.UserProfiles.SetUserProfileProperty
````

The application permissions for "PnP Management Shell" seems to be ok:

![image](https://user-images.githubusercontent.com/1762830/107927078-d082f700-6f76-11eb-9044-21b5778792b7.png)
